Docs Fleet add section about RollingUpdate fields (#1228)
googleforgames/agones: site/content/en/docs/Getting Started/create-fleetautoscaler.md

```

{{< alert title="Note" color="info">}}
If you want to update a `Fleet` which has `RollingUpdate` replacement strategy and is controlled by a `FleetAutoscaler`:

> 1. With `kubectl apply`: you should omit `replicas` parameter in a `Fleet` Spec before creating this `Fleet`.

Reviewer: ```suggestion
1. With `kubectl apply`: you should omit `replicas` parameter in a `Fleet` Spec before re-applying the `Fleet` configuration.
```

Author: Used your suggestion.

> 1. With `kubectl edit`: you should not change `replicas` parameter in a `Fleet` Spec when updating other field parameter.

Reviewer: ```suggestion
1. With `kubectl edit`: you should not change the `replicas` parameter in the `Fleet` Spec when updating other field parameters.
```

> If you follow both rules above, then `maxSurge` and `maxUnavaiable` parameters would be used and update would actually be a RollingUpdate.

Reviewer: ```suggestion
If you follow the rules above, then `maxSurge` and `maxUnavaiable` parameters will be used as the RollingUpdate strategy updates your Fleet.
```

> Otherwise fleet would be scaled according to Fleet `replicas` parameter first and only after certain amount of time it would be rescaled to fit `FleetAutoscaler` `BufferSize` parameter.

Reviewer: ```suggestion
Otherwise the Fleet would be scaled according to Fleet `replicas` parameter first and only after certain amount of time it would be rescaled to fit `FleetAutoscaler` `BufferSize` parameter.
```

> You could also check the behaviour of the Fleet with Fleetautoscaler on a test `Fleet` to see what you would get on your production environment.

Reviewer: ```suggestion
You could also check the behaviour of the Fleet with Fleetautoscaler on a test `Fleet` to preview what would occur on your production environment.
```
